decidim-proposals/app/views/decidim/proposals/proposals/show.html.erb

```diff
@@ -11,11 +11,21 @@
     <div class="author-data__main">
       <div class="author author--inline">
         <span class="author__avatar">
-          <%= image_tag @proposal.author_avatar_url %>
+          <% if @proposal.author&.deleted? %>
+            <%= image_tag ActionController::Base.helpers.asset_path("decidim/default-avatar.svg") %>
```

Comment:
Same, changing the avatar when deleting the user will make you skip this logic in the views